packages/grafana-ui/src/components/Select/SelectBase.test.tsx

```diff
@@ -43,6 +44,26 @@ describe('SelectBase', () => {
     expect(screen.getByLabelText('My select')).toBeInTheDocument();
   });
 
+  it('allows the value to be unset', async () => {
+    const Test = () => {
+      const option = { value: 'test-value', label: 'Test label' };
+      const [value, setValue] = useState<SelectableValue<string> | null>(option);
+
+      return (
+        <>
+          <button onClick={() => setValue(null)}>clear value</button>
+          <SelectBase value={value} onChange={setValue} options={[option]} />
+        </>
+      );
+    };
+
+    render(<Test />);
+    expect(screen.queryByText('Test label')).toBeInTheDocument();
+
+    await act(() => userEvent.click(screen.getByText('clear value')));
```

Comment:
```suggestion
    userEvent.click(screen.getByText('clear value'));
```
I think all user events are already wrapped in `act`, so it shouldn't be needed here.